machine_test.go

```diff
@@ -231,25 +231,8 @@ func TestJailerMicroVMExecution(t *testing.T) {
 		}
 	}
 
-	jailerBin := defaultJailerBinary
-	if _, err := os.Stat(filepath.Join(testDataPath, defaultJailerBinary)); err == nil {
-		jailerBin = filepath.Join(testDataPath, defaultJailerBinary)
-	}
-
 	ctx := context.Background()
-	cmd := NewJailerCommandBuilder().
-		WithBin(jailerBin).
-		WithGID(IntValue(cfg.JailerCfg.GID)).
-		WithUID(IntValue(cfg.JailerCfg.UID)).
-		WithNumaNode(IntValue(cfg.JailerCfg.NumaNode)).
-		WithID(cfg.JailerCfg.ID).
-		WithChrootBaseDir(cfg.JailerCfg.ChrootBaseDir).
-		WithExecFile(cfg.JailerCfg.ExecFile).
-		WithStdout(os.Stdout).
-		WithStderr(os.Stderr).
-		Build(ctx)
```

Comment:
Why do you want to remove these lines?

Reply:
This was due to merging with #180 which adds the jailer path in the configuration. So it is no longer needed elsewhere, ie the builder.